packages/picasso.js/src/core/chart-components/axis/axis-node-builder.js

```diff
@@ -214,18 +230,30 @@ export default function nodeBuilder(isDiscrete) {
     const layered = state.labels.activeMode === 'layered';
     let majorTickNodes;
 
+    if (settings.isRadial) {
+      buildOpts.startAngle = settings.startAngle;
+      buildOpts.endAngle = settings.endAngle;
+      buildOpts.radius = settings.radius;
```

Comment:
These new properties should be added in the specification, so they should be under the continous axis here:
https://github.com/qlik-oss/picasso.js/blob/4b1861a0c44c95a1d73b2d0d7f9ba9283ff32294/packages/picasso.js/src/core/chart-components/axis/axis-default-settings.js#L210

Reply:
Yes. I am wondering if I should create a new object with these properties (buildOpts.radial.startAngle for example), or if I should add them as a property of the buildOpts-object (buildOpts.startAngle).

Reply:
Fixed